Formula/pulumi.rb

```diff
@@ -0,0 +1,37 @@
+class Pulumi < Formula
+  desc "Define cloud applications and infrastructure in your favorite language"
+  homepage "https://pulumi.io/"
+  url "https://github.com/pulumi/pulumi.git",
+      :tag => "v0.14.3"
+
+  depends_on "dep" => :build
+  depends_on "go" => :build
+  depends_on "node@8" => :build
+  depends_on "pandoc" => :build
+  depends_on "pipenv" => :build
+  depends_on "python@2" => :build
+  depends_on "yarn" => :build
+
+  def install
+    ENV.deparallelize
+    ENV["GOPATH"] = buildpath
+    # Remove once opt/pulumi/bin is not needed anymore
+    ENV["PULUMI_ROOT"] = buildpath / "opt/pulumi/bin"
```

Comment:
Please cull the excessive spacing. `buildpath/"opt/pulumi/bin"` and so on will work just fine.